java/rocksjni/checkpoint.cc

```diff
@@ -66,3 +66,39 @@ void Java_org_rocksdb_Checkpoint_createCheckpoint(JNIEnv* env, jobject /*jobj*/,
     ROCKSDB_NAMESPACE::RocksDBExceptionJni::ThrowNew(env, s);
   }
 }
+
+/*
+ * Class:     org_rocksdb_ExportColumnFamily
+ * Method:    exportColumnFamily
+ * Signature: (JJLjava/lang/String;)J
+ */
+jlong Java_org_rocksdb_Checkpoint_exportColumnFamily(JNIEnv* env,
+                                                     jobject /*jobj*/,
```

Comment:
Could you remove the long-indenting? I think if you run `make format` it will correct this for you.

Could you also remove the commented out unused arg names like: ` /*jobj*/`?

This occurs in a few different places in the code.